### PR TITLE
Add Kafka operator docs (2.1.1)

### PR DESCRIPTION
**Pull request by cbuto**
## Jira Ticket

https://jira.d2iq.com/browse/D2IQ-82301

## Description of changes being made

Adding docs about the kafka operator in the workspace catalog and kafka cluster deployments in projects.

## Checklist

- [ ]  Copy this page to 2.2 (needs #3956)
- [ ] Update link to ZooKeeper operator docs once they are merged

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka/index.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    ...
+    ```
+
+1.  Apply the `KafkaCluster` to your project's namespace:
+
+    ```bash
+    kubectl apply -n ${PROJECT_NAMESPACE} -f simplekafkacluster.yaml
+    ```
+
+1.  Now that you have both the ZooKeeper cluster and Kafka cluster running in your project's namespace, you can find information on how to test and verify they are working as expected in the [Kafka Operator documentation](https://banzaicloud.com/docs/supertubes/kafka-operator/test/)
```
Comment (cneth) [inline pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka/index.md:71]: This is great!   However, don't you have to modify that command in the example to change the --zookeeper parameter to:  `zookeeper-client.<project namespace>:2181`?   Maybe we could mention that here?

```suggestion
1.  Now that you have both the ZooKeeper cluster and Kafka cluster running in your project's namespace, you can find information on how to test and verify they are working as expected in the [Kafka Operator documentation](https://banzaicloud.com/docs/supertubes/kafka-operator/test/).  When performing those steps, make sure to substitute: `zookeeper-client.<project namespace>:2181` anywhere that the zookeeper client address is mentioned.   
```

File: pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka/index.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    kubectl get kafkaclusters -A
+    kubectl get kafkausers -A
+    Kubectl get kafkatopics -A
+    ```
+
+1.  Deleting a `KafkaCluster` example:
+
+    ```bash
+    kubectl -n ${PROJECT_NAMESPACE} delete kafkacluster <name of KafkaCluster>
+    ```
```
Comment (cneth) [inline pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka/index.md:87]: Same comment here as on the Zookeeper topic, would be good to have the resources section here to make it a little easier for folks.

**Review by cneth — APPROVED** [review]

File: pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka/index.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+---
+
+## Getting Started
+
+<!-- TODO: Fix ZooKeeper operator link - this relies on the zookeeper docs being available in order to pass pre-commit -->
+To get started with creating and managing a Kafka Cluster in a project, you first need to deploy the [Kafka operator](../../../../../workspaces/applications/catalog-applications/kafka-operator/) and the [ZooKeeper operator](../../../../../workspaces/applications/catalog-applications/) in the workspace where the project exists.
+
+Once you have the Kafka operator deployed, you can create Kafka Clusters by applying a `KafkaCluster` custom resource in a project's namespace. You can find various examples of the custom resources and their configurations in the [Kafka operator repository](https://github.com/banzaicloud/koperator/tree/master/config/samples).
+
+## Example Deployment
```
Comment (msdolbey) [inline pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka/index.md:16]: Sentence case on headings

```suggestion
## Example deployment
```

File: pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka/index.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+1.  View all Kafka resources in the cluster:
+
+    ```bash
+    kubectl get kafkaclusters -A
+    kubectl get kafkausers -A
+    Kubectl get kafkatopics -A
+    ```
+
+1.  Deleting a `KafkaCluster` example:
```
Comment (msdolbey) [inline pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka/index.md:83]:
```suggestion
1.  Delete a `KafkaCluster` example:
```

File: pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka/index.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+
+1.  Apply the `KafkaCluster` to your project's namespace:
+
+    ```bash
+    kubectl apply -n ${PROJECT_NAMESPACE} -f simplekafkacluster.yaml
+    ```
+
+1.  Now that you have both the ZooKeeper cluster and Kafka cluster running in your project's namespace, you can find information on how to test and verify they are working as expected in the [Kafka Operator documentation](https://banzaicloud.com/docs/supertubes/kafka-operator/test/)
+
+## Deleting Kafka Custom Resources
```
Comment (msdolbey) [inline pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka/index.md:73]: Sentence case is all

```suggestion
## Deleting Kafka custom resources
```

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/kafka-operator/index.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+layout: layout.pug
+navigationTitle: Kafka Operator
+title: Kafka Operator
+menuWeight: 20
+excerpt: Information about the Kafka Operator
+---
+
+## Overview
+
+[Apache Kafka](https://kafka.apache.org/) is an open-source distributed event streaming platform used for high-performance data pipelines, streaming analytics, data integration, and mission-critical applications. The [Kafka Operator](https://banzaicloud.com/docs/supertubes/kafka-operator/) is a Kubernetes operator to automate provisioning, management, autoscaling, and operations of Apache Kafka clusters deployed to Kubernetes. It works by watching custom resources, such as `KafkaClusters`, `KafkaUsers`, and `KafkaTopics` to provision underlying Kubernetes resources (i.e `StatefulSets`) required for a production-ready Kafka Cluster.
```
Comment (msdolbey) [inline pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/kafka-operator/index.md:11]: grammar nit

```suggestion
[Apache Kafka](https://kafka.apache.org/) is an open-source distributed event streaming platform used for high-performance data pipelines, streaming analytics, data integration, and mission-critical applications. The [Kafka Operator](https://banzaicloud.com/docs/supertubes/kafka-operator/) is a Kubernetes operator to automate provisioning, management, autoscaling, and operations of Apache Kafka clusters deployed to Kubernetes. It works by watching custom resources, such as `KafkaClusters`, `KafkaUsers`, and `KafkaTopics`, to provision underlying Kubernetes resources (i.e `StatefulSets`) required for a production-ready Kafka Cluster.
```

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/kafka-operator/index.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+    kubectl get apps -n ${WORKSPACE_NAMESPACE}
+    ```
+
+For details on custom configuration for the operator, please refer to the [Kafka operator Helm Chart documentation](https://github.com/banzaicloud/koperator/tree/master/charts/kafka-operator#configuration).
+
+## Uninstall via the CLI
+
+Uninstalling the Kafka operator will not affect existing `KafkaCluster` deployments. After uninstalling the operator, you will need to manually remove any leftover Custom Resource Definitions (CRDs) from the operator.
+
+1.  Delete all Kafka custom resources that are deployed, see [Deleting Kafka Custom Resources](../../../../projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka#deleting-kafka-custom-resources).
```
Comment (msdolbey) [inline pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/kafka-operator/index.md:29]:
```suggestion
1.  Delete all Kafka custom resources that are deployed (see [Deleting Kafka Custom Resources](../../../../projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka#deleting-kafka-custom-resources)).
```

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/kafka-operator/index.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+
+1.  Uninstalling a Kafka operator `AppDeployment`:
```
Comment (msdolbey) [inline pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/kafka-operator/index.md:31]:
```suggestion
1.  Uninstall a Kafka operator `AppDeployment`:
```

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/kafka-operator/index.md [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+
+    ```bash
+    kubectl -n <workspace namespace> delete AppDeployment <name of AppDeployment>
+    ```
+
+1.  Removing Kafka CRDs:
```
Comment (msdolbey) [inline pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/kafka-operator/index.md:37]:
```suggestion
1.  Remove Kafka CRDs:
```

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/kafka-operator/index.md [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+    <p class="message--note"><strong>NOTE: </strong>the CRDs are not finalized for deletion until you delete the associated custom resources.</p>
```
Comment (msdolbey) [inline pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/kafka-operator/index.md:38]:
```suggestion
    <p class="message--note"><strong>NOTE: </strong>The CRDs are not finalized for deletion until you delete the associated custom resources.</p>
```

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/kafka-operator/index.md [hunk 11]
```diff
@@ -0,0 +1,49 @@
...
+    ```bash
+    kubectl delete crds kafkaclusters.kafka.banzaicloud.io kafkausers.kafka.banzaicloud.io kafkatopics.kafka.banzaicloud.io
+    ```
+
+## Resources
+
+- [Kafka Operator Documentation](https://banzaicloud.com/docs/supertubes/kafka-operator/)
+- [Kafka Operator GitHub Repository](https://github.com/banzaicloud/koperator)
+- [Sample Kafka Operator Custom Resources](https://github.com/banzaicloud/koperator/tree/master/config/samples)
+- [Apache Kafka Documentation](https://kafka.apache.org/)
```
Comment (msdolbey) [inline pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/kafka-operator/index.md:49]: Probably doesn't make too huge of a difference, but this link goes straight to their docs. Also, not related, but the Kafka Operator for short goes by Koperator, which is just funny to me.

```suggestion
- [Apache Kafka Documentation](https://kafka.apache.org/documentation/)
```

**Review by msdolbey — APPROVED** [review]
Some small nits, but looks good to me other than that.
👍 
Kafka 1.0

File: pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka/index.md [hunk 12]
```diff
@@ -0,0 +1,87 @@
...
+menuWeight: 5
+excerpt: Deploying Kafka in a project
+---
+
+## Getting Started
+
+<!-- TODO: Fix ZooKeeper operator link - this relies on the zookeeper docs being available in order to pass pre-commit -->
+To get started with creating and managing a Kafka Cluster in a project, you first need to deploy the [Kafka operator](../../../../../workspaces/applications/catalog-applications/kafka-operator/) and the [ZooKeeper operator](../../../../../workspaces/applications/catalog-applications/) in the workspace where the project exists.
+
+Once you have the Kafka operator deployed, you can create Kafka Clusters by applying a `KafkaCluster` custom resource in a project's namespace. You can find various examples of the custom resources and their configurations in the [Kafka operator repository](https://github.com/banzaicloud/koperator/tree/master/config/samples).
```
Comment (Colin1841) [inline pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka/index.md:14]:
```suggestion
After you deploy the Kafka operator deployed, create Kafka Clusters by applying a `KafkaCluster` custom resource in a project's namespace. Refer to the the [Kafka operator repository](https://github.com/banzaicloud/koperator/tree/master/config/samples) for examples of the custom resources and their configurations.
```

File: pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka/index.md (hunk 3 above)
Comment (Colin1841) [reply pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka/index.md:16]: Best practice is always to provide an introduction of the what/why variety, before a procedure, even when it's obvious. So something like "This example deployment provides you with xxx, and leaves you with yyy," A sentence or two is all we usually need.

File: pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka/index.md (hunk 1 above)
Comment (Colin1841) [reply pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka/index.md:71]: For the "this procedure results in this..." kind of statements, lets remove the step heading and just go with straight text. We want to reserve numbered action items only for the steps a user takes.

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/kafka-operator/index.md [hunk 13]
```diff
@@ -0,0 +1,49 @@
...
+menuWeight: 20
+excerpt: Information about the Kafka Operator
+---
+
+## Overview
+
+[Apache Kafka](https://kafka.apache.org/) is an open-source distributed event streaming platform used for high-performance data pipelines, streaming analytics, data integration, and mission-critical applications. The [Kafka Operator](https://banzaicloud.com/docs/supertubes/kafka-operator/) is a Kubernetes operator to automate provisioning, management, autoscaling, and operations of Apache Kafka clusters deployed to Kubernetes. It works by watching custom resources, such as `KafkaClusters`, `KafkaUsers`, and `KafkaTopics` to provision underlying Kubernetes resources (i.e `StatefulSets`) required for a production-ready Kafka Cluster.
+
+## Install
+
```
Comment (Colin1841) [inline pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/kafka-operator/index.md:14]: Same as commented above; an introductory sentence or two is always helpful.

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/kafka-operator/index.md [hunk 14]
```diff
@@ -0,0 +1,49 @@
...
+1.  Follow the generic installation instructions for workspace catalog applications on the [Application Deployment](../application-deployment/) page.
+
+1.  Within the `AppDeployment`, update the `appRef` to specify the correct `kafka-operator` App. You can find the `appRef.name` by listing the available `Apps` in the workspace namespace:
+
+    ```bash
+    kubectl get apps -n ${WORKSPACE_NAMESPACE}
+    ```
+
+For details on custom configuration for the operator, please refer to the [Kafka operator Helm Chart documentation](https://github.com/banzaicloud/koperator/tree/master/charts/kafka-operator#configuration).
```
Comment (Colin1841) [inline pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/kafka-operator/index.md:23]:
```suggestion
Refer to the [Kafka operator Helm Chart documentation](https://github.com/banzaicloud/koperator/tree/master/charts/kafka-operator#configuration) for details on custom configuration for the operator.
```

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/kafka-operator/index.md [hunk 15]
```diff
@@ -0,0 +1,49 @@
...
+
+## Uninstall via the CLI
+
+Uninstalling the Kafka operator will not affect existing `KafkaCluster` deployments. After uninstalling the operator, you will need to manually remove any leftover Custom Resource Definitions (CRDs) from the operator.
```
Comment (Colin1841) [inline pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/kafka-operator/index.md:27]:
```suggestion
Uninstalling the Kafka operator does not affect existing `KafkaCluster` deployments. After uninstalling the operator, you must manually remove any remaining Custom Resource Definitions (CRDs) from the operator.
```

File: pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka/index.md (hunk 1 above)
Comment (cbuto) [reply pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka/index.md]: thanks @Colin1841!  @cneth this should be fixed now 🙏

File: pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka/index.md (hunk 2 above)
Comment (cbuto) [reply pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka/index.md:101]: fixed in 2fcad2ad6824e86564597d3ab4a8c318b7102fda

File: pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka/index.md (hunk 3 above)
Comment (cbuto) [reply pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka/index.md]: @msdolbey @Colin1841 fixed in 2fcad2ad6824e86564597d3ab4a8c318b7102fda

File: pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka/index.md (hunk 4 above)
Comment (cbuto) [reply pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka/index.md]: fixed in 2fcad2ad6824e86564597d3ab4a8c318b7102fda

File: pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka/index.md (hunk 5 above)
Comment (cbuto) [reply pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka/index.md]: 💼 fixed in 2fcad2ad6824e86564597d3ab4a8c318b7102fda

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/kafka-operator/index.md (hunk 6 above)
Comment (cbuto) [reply pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/kafka-operator/index.md]: fixed in 2fcad2ad6824e86564597d3ab4a8c318b7102fda

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/kafka-operator/index.md (hunk 7 above)
Comment (cbuto) [reply pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/kafka-operator/index.md]: fixed in 2fcad2ad6824e86564597d3ab4a8c318b7102fda

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/kafka-operator/index.md (hunk 8 above)
Comment (cbuto) [reply pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/kafka-operator/index.md]: fixed in 2fcad2ad6824e86564597d3ab4a8c318b7102fda

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/kafka-operator/index.md (hunk 9 above)
Comment (cbuto) [reply pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/kafka-operator/index.md]: fixed in 2fcad2ad6824e86564597d3ab4a8c318b7102fda

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/kafka-operator/index.md (hunk 10 above)
Comment (cbuto) [reply pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/kafka-operator/index.md]: fixed in 2fcad2ad6824e86564597d3ab4a8c318b7102fda

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/kafka-operator/index.md (hunk 11 above)
Comment (cbuto) [reply pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/kafka-operator/index.md]: good call, fixed in 2fcad2ad6824e86564597d3ab4a8c318b7102fda. 😆

File: pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka/index.md (hunk 12 above)
Comment (cbuto) [reply pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka/index.md]: fixed in 2fcad2ad6824e86564597d3ab4a8c318b7102fda

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/kafka-operator/index.md (hunk 14 above)
Comment (cbuto) [reply pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/kafka-operator/index.md]: fixed in 2fcad2ad6824e86564597d3ab4a8c318b7102fda

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/kafka-operator/index.md (hunk 15 above)
Comment (cbuto) [reply pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/kafka-operator/index.md]: fixed in 2fcad2ad6824e86564597d3ab4a8c318b7102fda

**Review by cwyl02 — APPROVED** [review]

File: pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka/index.md [hunk 16]
```diff
@@ -0,0 +1,108 @@
...
+menuWeight: 5
+excerpt: Deploying Kafka in a project
+---
+
+## Getting started
+
+<!-- TODO: Fix ZooKeeper operator link - this relies on the zookeeper docs being available in order to pass pre-commit -->
+To get started with creating and managing a Kafka Cluster in a project, you first need to deploy the [Kafka operator](../../../../../workspaces/applications/catalog-applications/kafka-operator/) and the [ZooKeeper operator](../../../../../workspaces/applications/catalog-applications/) in the workspace where the project exists.
+
+After you deploy the Kafka operator, create Kafka Clusters by applying a `KafkaCluster` custom resource in a project's namespace. Refer to the [Kafka operator repository](https://github.com/banzaicloud/koperator/tree/master/config/samples) for examples of the custom resources and their configurations.
```
Comment (Colin1841) [inline pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka/index.md:14]:
```suggestion
After deploying the Kafka operator, create Kafka Clusters by applying a `KafkaCluster` custom resource in a project's namespace. Refer to the [Kafka operator repository](https://github.com/banzaicloud/koperator/tree/master/config/samples) for examples of the custom resources and their configurations.
```

File: pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka/index.md [hunk 17]
```diff
@@ -0,0 +1,108 @@
...
+
+## Example deployment
+
+This example deployment walks you through first deploying a ZooKeeper cluster and then a Kafka cluster in a project namespace. After following the steps below, you should have a running ZooKeeper cluster and Kafka cluster ready for use in your project's namespace.
```
Comment (Colin1841) [inline pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka/index.md:18]:
```suggestion
This example deployment walks you through first deploying a ZooKeeper cluster and then a Kafka cluster in a project namespace. The result of this procedure is a running ZooKeeper cluster and Kafka cluster ready for use in your project's namespace.
```

File: pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka/index.md [hunk 18]
```diff
@@ -0,0 +1,108 @@
...
+        ```
+
+    -   GNU sed
+
+        ```bash
+        # If you are using GNU sed
+        sed -i "s/zookeeper-client.zookeeper:2181/zookeeper-client.${PROJECT_NAMESPACE}:2181/g" simplekafkacluster.yaml
+        ```
+
+1.  The file should now contain the following line:
```
Comment (Colin1841) [inline pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka/index.md:64]:
```suggestion
1.  Verify the file contains the following line:
```

File: pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka/index.md [hunk 19]
```diff
@@ -0,0 +1,108 @@
...
+1.  Check the status of your Kafka cluster using `kubectl`:
+
+    ```bash
+    kubectl -n ${PROJECT_NAMESPACE} get kafkaclusters 
+    ```
+
+With both the ZooKeeper cluster and Kafka cluster running in your project's namespace, refer to the [Kafka Operator documentation](https://banzaicloud.com/docs/supertubes/kafka-operator/test/) for information on how to test and verify they are working as expected in. When performing those steps, ensure you substitute: `zookeeper-client.<project namespace>:2181` anywhere that the zookeeper client address is mentioned.
+
+## Deleting Kafka custom resources
+
```
Comment (Colin1841) [inline pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka/index.md:88]:
```suggestion

Follow these steps to delete the Kafka custom resources.

```

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/kafka-operator/index.md [hunk 20]
```diff
@@ -0,0 +1,51 @@
...
+excerpt: Information about the Kafka Operator
+---
+
+## Overview
+
+[Apache Kafka](https://kafka.apache.org/) is an open-source distributed event streaming platform used for high-performance data pipelines, streaming analytics, data integration, and mission-critical applications. The [Kafka Operator](https://banzaicloud.com/docs/supertubes/kafka-operator/) is a Kubernetes operator to automate provisioning, management, autoscaling, and operations of Apache Kafka clusters deployed to Kubernetes. It works by watching custom resources, such as `KafkaClusters`, `KafkaUsers`, and `KafkaTopics`, to provision underlying Kubernetes resources (i.e `StatefulSets`) required for a production-ready Kafka Cluster.
+
+## Install
+
+These install instructions describe how to install the Kafka operator in a workspace. After following the install instructions, you will have the Kafka operator running in a workspace namespace, ready to manage and create Kafka clusters in any project namespaces. See the [Kafka custom resource documentation](../../../../projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka/) for more information on creating Kafka clusters.
```
Comment (Colin1841) [inline pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/kafka-operator/index.md:15]:
```suggestion
Follow these steps to install the Kafka operator in a workspace. This procedure results in a Kafka operator running in a workspace namespace, ready to manage and create Kafka clusters in any project namespaces. See the [Kafka custom resource documentation](../../../../projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka/) for more information on creating Kafka clusters.
```

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/kafka-operator/index.md [hunk 21]
```diff
@@ -0,0 +1,51 @@
...
+    kubectl get apps -n ${WORKSPACE_NAMESPACE}
+    ```
+
+Refer to the [Kafka operator Helm Chart documentation](https://github.com/banzaicloud/koperator/tree/master/charts/kafka-operator#configuration) for details on custom configuration for the operator.
+
+## Uninstall via the CLI
+
+Uninstalling the Kafka operator does not affect existing `KafkaCluster` deployments. After uninstalling the operator, you must manually remove any remaining Custom Resource Definitions (CRDs) from the operator.
+
+1.  Delete all Kafka custom resources that are deployed (see [Deleting Kafka Custom Resources](../../../../projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka#deleting-kafka-custom-resources)).
```
Comment (Colin1841) [inline pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/kafka-operator/index.md:31]:
```suggestion
1.  Delete all of the deployed Kafka custom resources (see [Deleting Kafka Custom Resources](../../../../projects/applications/catalog-applications/custom-resources-workspace-catalog/kafka#deleting-kafka-custom-resources)).
```

**Review by Colin1841 — APPROVED** [review]

**Pull request merged**